jenkins/agent-ctl.sh

```diff
@@ -52,15 +52,15 @@ usage() {
 set -o nounset
 set -o errexit
 
-DOCKER_VERSION='1.9.1-0~wheezy'
+DOCKER_VERSION='1.9.1-0~jessie'
```

Comment:
That is outside the scope of this PR.

Updating to debian8 should be a noop. Updating docker -- especially since we still use 1.9 in our [test images](https://github.com/kubernetes/test-infra/blob/master/jenkins/test-image/Dockerfile#L48) -- seems like a substantially more risky change.

This should happen once we have test images using different docker versions.